### PR TITLE
ugps: update to latest HEAD, introduce baudrate

### PR DESCRIPTION
**Pull request by db4rne**
Build-tested: ramips/ltap-2hnd
Run-tested: ramips/ltap-2hnd

### CODE REVIEW

File: package/utils/ugps/files/ugps.init [hunk 1]
```diff
@@ -12,6 +12,7 @@ service_triggers() {
 
 start_service() {
 	local tty="$(uci get gps.@gps[-1].tty)"
+	local baudrate="$(uci get gps.@gps[-1].baudrate)"
```
Comment (obsy) [inline package/utils/ugps/files/ugps.init:15]: What if baudrate not defined?

```
local baudrate="$(uci get gps.@gps[-1].baudrate) || echo 0"
```

?

Comment (obsy) [reply package/utils/ugps/files/ugps.init:15]: Wrong, sh: out of range is appear. See debug with empty baudrate:
```
#!/bin/sh

start_service()
{
        local baudrate="$(uci get gps.@gps[-1].baudrate)"
        [ "$baudrate" -eq 0 ] || echo procd_append_param command "-b ${baudrate}"
}

start_service
```

```
# sh -x ./test1.sh
+ start_service
+ uci get 'gps.@gps[-1].baudrate'
uci: Entry not found
+ local 'baudrate='
+ '['  -eq 0 ]
sh: out of range
+ echo procd_append_param command '-b '
procd_append_param command -b
```

Comment (db4rne) [reply package/utils/ugps/files/ugps.init:15]: oh, makes sense, thanks.

**Comment by PolynomialDivision** [comment]
Please, add short log when bumping the package in the git commit message, like we do in other commits:
https://github.com/openwrt/openwrt/commit/063641f8cf7990731919b950c1bf9eb15d1e74c9

**Comment by blogic** [comment]
LGTM

**Pull request merged**